holoviews/core/spaces.py

```diff
@@ -509,6 +516,47 @@ def get_nested_streams(dmap):
     return list(set(layer_streams))
 
 
+
+def get_stream_sources(obj, branch=0):
```

Comment:
I'm thinking this function should really be called something like ``overlayable_zorder``.